### PR TITLE
Iv 4861 rhel subscription

### PR DESCRIPTION
**Pull request by lopaka**

### CODE REVIEW

File: rll-examples/redhat-subscription-unregister.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  fi
+fi
+
+if [[ "$ID" != "rhel" ]]; then
+  echo "RedHat Subscription Management is only used by RedHat Linux"
+  exit 0
+fi
+
+# Check if server is already registered
+if sudo subscription-manager identity; then
```
Comment (douglaswth) [inline rll-examples/redhat-subscription-unregister.sh]: This script should probably be checking the `DECOM_REASON` variable to determine if it should actually be taking an action.

File: rll/redhat-subscription-register.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+#     Advanced: true
+# Attachments: []
+# ...
+
+set -e
+
+# Read/source os-release to obtain variable values determining OS
+if [[ -e /etc/os-release ]]; then
+  source /etc/os-release
+else
```
Comment (psschroeter) [inline rll/redhat-subscription-register.sh]: why not just an elif?

**Review by psschroeter — APPROVED** [review]
LGTM

**Review by douglaswth — APPROVED** [review]
LGTM

**Pull request merged**